lib/rubocop/lsp/diagnostic.rb

```diff
@@ -184,6 +171,18 @@ def ensure_uri_scheme(uri)
         uri.scheme = 'file' if uri.scheme.nil?
         uri
       end
+
+      def char_pos(nchars = nil)
```

Comment:
It seems better to take this opportunity to use an unabbreviated name instead of `character_position`. The argument `nchars` should also be given a non-abbreviated name.

Reply:
Changed to `to_position_character(utf8_index)`